src/diceBot/ShadowRun5.rb

```diff
@@ -35,7 +35,7 @@ def rollDiceCommand(command)
     m = /(\d+B6)@(\d+)/.match(command)
     b_dice = m[1]
     limit = m[2].to_i
-    output_before_limited, secret = bcdice.checkBDice(b_dice)
+    output_before_limited, = bcdice.checkBDice(b_dice)
```

Comment:
ここでは1つの値のみ返す `BCDice#bdice` が使えます。

```suggestion
    output_before_limited = bcdice.bdice(b_dice)
```